.github/workflows/go.yml

```diff
@@ -21,6 +21,23 @@ jobs:
       with:
         go-version: '1.20'
 
+    - name: Enforce gofmt
+      run: |
+        test -z $(gofmt -l cmd)
```

Comment:
This is harder than it needs to be because of `vendor`, which has a lot of violations  - could also reverse it and test everything but filter out vendor results.